pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -82,6 +84,19 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 // Name returns Name for the precondition.
 func (pf *Upgradeable) Name() string { return "ClusterVersionUpgradeable" }
 
+// getCurrentVersion determines and returns the cluster's current version by iterating through the
+// provided update history until it finds the first version with update State of Completed. If a
+// Completed version is not found the last entry's version is returned.
```

Comment:
nit: "last entry's" seems like it could be misread as "most recent".  Can we say:

> If a Completed version is not found, the originally installed version is returned.

Reply:
Fixed in [a21d201]( https://github.com/openshift/cluster-version-operator/pull/394/commits/a21d201ef9565b027ae759baac16c48d20f68eca)